docs/Networking.asciidoc

```diff
@@ -48,235 +48,275 @@ automatically computed from $worker_id if not set.
 In TAP mode the system administrator is expected to configure the
 network, required internet access etc on the host manually.
 
-TAP devices need be owned by the +_openqa-worker+ user for openQA to
-be able to access them.
+=== VDE Based Network
 
-[source,sh]
----------------
-tunctl -u _openqa-worker -p -t tap0
----------------
+Virtual Distributed Ethernet provides a software switch that runs in
+user space. It allows to connect several qemu instances without
+affecting the system's network configuration.
 
-If you want to use TAP device which doesn't exist on the system,
-you need to set CAP_NET_ADMIN capability on qemu binary file:
+The openQA workers need a vde_switch instance running. The workers
+reconfigure the switch as needed by the job.
 
-[source,sh]
----------------
-zypper in libcap-progs
-setcap CAP_NET_ADMIN=ep /usr/bin/qemu-system-x86_64
----------------
+==== Basic, single machine tests
+
+To start with a basic configuration like qemu user mode networking,
+create a machine with the following settings:
+
+- +VDE_SOCKETDIR=/run/openqa+
+- +NICTYPE=vde+
+- +NICVLAN=0+
 
-Network setup can be changed after qemu is started using network configure script
-specified in TAPSCRIPT variable.
+Start switch and user mode networking:
 
-Sample script to add TAP device to existing bridge br0:
 [source,sh]
 ---------------
-sudo brctl addif br0 $1
-sudo ip link set $1 up
+systemctl start openqa-vde_switch
+systemctl start openqa-slirpvde
 ---------------
 
-== TAP with Open vSwitch
+With this setting all jobs on the same host would be in the same
+network share the same SLIRP instance though.
+
+== Multi Machine Tests Setup
 
-The recommended way to configure the network for TAP devices is using Open vSwitch.
-There is a support service +os-autoinst-openvswitch.service+ which sets vlan number
-of Open vSwitch ports based on +NICVLAN+ variable - this separates the groups of
-tests from each other.
+The section provides steps for setting up openQA environment to run tests that require
+network connection between several machines (e.g. client -- server tests)
+
+The configuration of TAP interfaces will use _Open vSwitch_ for virtual
+switch, _SuSEfirewall2_ for NAT and _wicked_ as network manager.
```

Comment:
The problem with *wicked* is that this is an openSUSE specific solution so should be handled as such. E.g. mention it as an *example* applicable for openSUSE.

Reply:
Ok, will change that part.